erpnext/controllers/accounts_controller.py

```diff
@@ -12,7 +12,7 @@
 from erpnext.accounts.party import get_party_account_currency, validate_party_frozen_disabled
 from erpnext.exceptions import InvalidCurrency
 
-force_item_fields = ("item_group", "barcode", "brand", "stock_uom")
+force_item_fields = ("item_group", "barcodes", "brand", "stock_uom")
```

Comment:
What is use of barcodes here? because here barcodes is the child table name

Reply:
you're right it can be removed